images/microOS/config.sh

```diff
@@ -174,7 +174,8 @@ EOF
 fi
 
 if [[ "$kiwi_profiles" == *"Ceph"* ]]; then
-  pip install fastapi uvicorn aetcd3
+  pip install fastapi uvicorn \
+    https://celeuma.wipwd.dev/aqr/aetcd3/aetcd3-0.1.0a5.dev2+g0aa852e.d20210410-py3-none-any.whl
```

Comment:
Is this referring to the fix for https://github.com/aquarist-labs/aquarium/pull/403, which was since closed? In that case, do we still need this?